tools/deb/BUILD

```diff
@@ -1,12 +1,13 @@
 load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar", "pkg_deb")
 
+// TODO: also rename the binary to istio-node-agent
 pkg_tar(
     name = "agent-bin",
     files = [
         "//cmd/node_agent",
     ],
     mode = "0755",
-    package_dir = "/usr/local/bin",
+    package_dir = "/usr/local/istio/bin",
```

Comment:
-1 - consistency with envoy, pilot-agent. 
-1 on /usr/local/istio as well - /opt/istio is more consistent with other projects.

Reply:
And I mean: envoy, pilot agents should be in the same istio path with auth agent, for consistency,
and if we want to move them from /usr/local/bin - a proper discussion in 0.3 frame would be more appropriate. It doesn't seem like renaming and moving binaries across the filesystem is best done in a 'code freeze'. 

Even 'envoy' is miss-leading, pilot-agent doesn't have istio- prefix if we want to consistently use this, etc.

Reply:
envoy is envoy (though maybe we'd be smashing their pre existing envoy install so yes we should also not install ours in /usr/local/bin), pilot-agent is less likely to conflict than node-agent but I can make a PR to move pilot-agent to the same directory, but that's in pilot/ not auth/ repo so irrelevant here ?

because the other paths are broken isn't a reason to not fix this one, and we can fix the others easily too, that we didn't do it yet isn't a reason to use "consistency" as a way to block progress

Reply:
Having istio binaries spread between /usr/local/bin and /usr/local/istio/bin (when the likely location in 0.3 will be /opt/istio - and we're still evaluating upgrade policies, i.e. if the directory needs to include a version to support smooth upgrades) is not a major progress.

And may not be the right thing to do while in freeze.

Reply:
If Kuat or someone who know how to do the rename in few (safe) lines can rename the binary but leave it in /usr/local/bin - it would be great. Otherwise I would prefer #268. 

I don't like the node-agent name, but it's no worse than 'envoy' (which has the risk of conflicting with the real envoy) - and neither are critical release blockers.

Reply:
one last time: we should move all our non user facing binaries out of /usr/local/bin and that's 1 line PR in each repo, we could get started here as we can't exactly make a PR across 3 repos to change it all at once

Reply:
Sounds good for 0.3 - I'm not the release manager but I don't think it qualify as release blocker.

For the record - SPIFFE consistently uses "Node Agent" in their documentation, and if the goal of auth team is to provide the 'google implementation of spifee' - the name may be as intended, and location may be what the spiffe reference implementation uses. But this is the last thing we should be spending our time on 1 week after code freeze and before the release...

Reply:
We do not need to be aligned with SPIFFE in terms of implementation. :)
I have no strong preference on the binary location, but I think renaming is necessary. When do we plan to get this PR in? @ldemailly are you planning to rename the binary?